test/node/timestamp.test.ts

```diff
@@ -153,4 +156,42 @@ describe('Timestamp', () => {
       }
     });
   });
+
+  describe('toLong()', function () {
+    const timestamp = new Timestamp({ t: 19324, i: 15 });
+    it('sets Long.low to Timestamp.i', function () {
+      expect(timestamp.toLong().low).to.equal(timestamp.i);
+    });
+    it('sets Long.high to Timestamp.t', function () {
+      expect(timestamp.toLong().high).to.equal(timestamp.t);
+    });
+    it('round trips correctly', function () {
+      expect(new Timestamp(timestamp.toLong())).to.deep.equal(timestamp);
+    });
+  });
+
+  describe('fromBits()', function () {
+    it('sets i and t correctly', function () {
+      const ts = Timestamp.fromBits(0, 1);
+      expect(ts.i).to.equal(0);
+      expect(ts.t).to.equal(1);
+    });
+  });
+
+  describe('static fromNumber()', function () {
+    describe('accepts', function () {});
+    describe('rejects', function () {});
+  });
```

Comment:
We don't have any tests that check the behaviour of `Long.fromNumber` (which this function calls), should we add any tests for it here or in long.test.ts?